pkg/reconciler/controllers/remediator_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC
```

Comment:
google policy is to leave the copyright at the original year the file was added.